tests/test_bootstrap_wizard.sh

```
@@ -0,0 +1,461 @@
+mkdir -p "$unknown_resume_target"
+set +e
+ur_out=$(BOOTSTRAP_SKIP_TOOL_CHECK=1 BOOTSTRAP_SKIP_MERGEPATH_GUARD=1 \
+         BOOTSTRAP_AUTO_CONFIRM=1 BOOTSTRAP_AUTO_PROMPT=skip \
+         "$SCRIPT" my-new-repo \
+         --target-dir "$unknown_resume_target" \
+         --description d --visibility private --firebase none \
+         --codex-app n --project new --dry-run \
+         --resume not-a-real-stage 2>&1)
+ur_ec=$?
+set -e
+[ "$ur_ec" -ne 0 ] && pass "--resume with unknown stage → non-zero" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)